auto_nag/scripts/old_p2_bug.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag import utils
+
+
+class oldP1Bug(BzCleaner):
+
+    def __init__(self):
+        super(oldP1Bug, self).__init__()
+        self.nweeks = utils.get_config(self.name(), 'number_of_weeks', 104)
```

Comment:
Add an entry in the config file: 'number_of_years'

Reply:
@calixteman Well, I was thinking about decreasing that to one year or a few months ago. This is why I kept weeks :)
wdyt?

Reply:
ok, so wfm